javascript/OMakefile

```diff
@@ -1,10 +1,7 @@
 
 .PHONY: noarch-install
 noarch-install:
-	mkdir -p $(DESTDIR)$(OPTDIR)/debug/www
-	cp *.js *.html *.css $(DESTDIR)$(OPTDIR)/debug/www
-	mkdir -p $(DESTDIR)$(OPTDIR)/debug/www/jquery
-	cp -v jquery/* $(DESTDIR)$(OPTDIR)/debug/www/jquery
-	mkdir -p $(DESTDIR)$(OPTDIR)/debug/www/images
-	cp -v images/xen_logo.gif $(DESTDIR)$(OPTDIR)/debug/www/images
+	echo "removed"
```

Comment:
Calling an install target and getting a "removed" is a surprise. How about "nothing to be done" or similar?